memberships/models.py

```diff
@@ -4,18 +4,34 @@
 from django.db import transaction
 from django.conf import settings
 from django.utils import timezone
-
 from .services import StripeGateway
 
 
 class Member(models.Model):
-    full_name = models.CharField(max_length=255)
-    preferred_name = models.CharField(max_length=255)
+    full_name = models.CharField(max_length=255, verbose_name="Full name")
+    preferred_name = models.CharField(max_length=255, verbose_name="Preferred name")
     email = models.EmailField(max_length=254)
-    birth_date = models.DateField()
+    birth_date = models.DateField(verbose_name="Date of birth")
     constitution_agreed = models.BooleanField()
     stripe_customer_id = models.CharField(max_length=255)
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+    profile_image = models.ImageField(upload_to="images/", blank=True, verbose_name="Selfie")
+    telephone = models.CharField(max_length=255, blank=True, verbose_name="Phone number")
+    minecraft_username = models.CharField(max_length=255, blank=True, verbose_name="Minecraft username")
+    address_1 = models.CharField(max_length=255, blank=True, verbose_name="Address line one")
+    address_postcode = models.CharField(max_length=10, blank=True, verbose_name="Postcode")
+    gift_aid = models.BooleanField(default=False, verbose_name="Gift aid")
+    gdpr_likeness = models.BooleanField(default=False, verbose_name="Likeness")
+    gdpr_sms_updates = models.BooleanField(default=False, verbose_name="SMS updates")
+    gdpr_sms_notifications = models.BooleanField(default=False, verbose_name="SMS notifications")
+    gdpr_email_updates = models.BooleanField(default=False, verbose_name="Email updates")
+    gdpr_email_notifications = models.BooleanField(default=False, verbose_name="Email notifications")
+    gdpr_telephone_updates = models.BooleanField(default=False, verbose_name="Phone updates")
+    gdpr_telephone_notifications = models.BooleanField(default=False, verbose_name="Phone notifications")
+    gdpr_post_updates = models.BooleanField(default=False, verbose_name="Post updates")
+    gdpr_post_notifications = models.BooleanField(default=False, verbose_name="Post notifications")
+    renewal_date = models.DateField()
```

Comment:
The `makemigrations` command is saying this needs a default value. Could we assign one or remove the field for now.

I would imagine member renewals might be more complex than this anyways so I would probably remove it if it were me.

Reply:
In fact we absolutely should remove this for now as I get this error when registering with it in place, so it's probably easier to just think about it later :+1: 

![image](https://user-images.githubusercontent.com/3310215/99082794-89bee300-25bc-11eb-800d-e85639886558.png)

Reply:
Fair cop - done